UIkitDZ/VC/LoginVC.swift

```diff
@@ -0,0 +1,95 @@
+//  UIkitDZ
+//
+//  Created by Timofey Privalov on 30.06.2021.
+//
+
+import UIKit
+
+class LoginVC: UIViewController {
+    
+    
+    var authorizeButton = UIButton()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        createLogoImage(img: "wendys.png", x: 100, y: 60)
+        
+        createLabel(nameType: "Phone number", x: 50, y: 230 + 50)
+        createLabel(nameType: "Password", x: 50, y: 330 + 50)
+        
+        createTextView(x: 50, y: 255 + 50, text: "+7800300100", type: 1)
+        lineAdd(x: 50, y: 305 + 50)
+        createTextView(x: 50, y: 355 + 50, text: "pass", type: 777)
+        lineAdd(x: 50, y: 405 + 50)
+        
+        createLoginButton()
+        
+    }
+    
+    // Mark: кнопка login
+    func createLoginButton() {
+        let authorizeButton = UIButton()
+        authorizeButton.setTitle("Авторизоваться", for: .normal)
+        authorizeButton.backgroundColor = .orange
+        authorizeButton.setTitleColor(.white, for: .normal)
+        authorizeButton.layer.borderColor = UIColor.orange.cgColor
+        authorizeButton.layer.borderWidth = 3
+        authorizeButton.layer.cornerRadius = 10
+        authorizeButton.frame = CGRect(x: 100, y: 550, width: 190, height: 50)
+        authorizeButton.addTarget(self, action: #selector(showFood), for: .touchUpInside)
+        self.view.addSubview(authorizeButton)
+    }
+    // -Mark: добавить полосу
+    func lineAdd(x: Int, y: Int) {
+        let line = UIView()
+        line.backgroundColor = UIColor.lightGray
+        line.frame = CGRect(x: x, y: y, width: 300, height: 1)
+        self.view.addSubview(line)
+    }
+    
+    // -Mark: текстовые поля navigation controller
+    func createTextView(x: Int, y: Int, text: String, type: Int) {
+        let textField = UITextField(frame: CGRect(x: x, y: y, width: 200, height: 60))
+        if type == 1 {
+            textField.keyboardType = .phonePad
+        } else {
+            textField.keyboardType = .numbersAndPunctuation
+        }
+        textField.placeholder = text
+        self.view.addSubview(textField)
+    }
+    
+    // -Mark: презентация navigation controller
+    @objc func showFood() {
+        let foodVC = FoodVC()
+        let navControllerFood = UINavigationController(rootViewController: foodVC)
+        navControllerFood.modalPresentationStyle = .fullScreen
+        self.present(navControllerFood, animated: true, completion: nil)
+    }
+    // Mark: добавить лэйблы  email, pass
+    func createLabel(nameType: String, x: Int, y: Int) {
+        let namePizza = UILabel()
+        namePizza.text = nameType
+        namePizza.textColor = UIColor.blue
+        namePizza.frame = CGRect(x: x, y: y, width: 250, height: 60)
+        namePizza.backgroundColor = UIColor.clear
+        namePizza.textColor = .blue
+        namePizza.font = namePizza.font.withSize(15)
+        self.view.addSubview(namePizza)
+    }
+    
+    // Mark: добавить лого
```

Comment:
```suggestion
    // Mark: добавить лого 
```
такие марки не по сайду смотри закреп